packages/v4/patternfly-docs/content/get-started/design.md

```diff
@@ -5,47 +5,45 @@ section: get-started
 ---
 import './get-started.css';
 
-Whether you are converting from a PatternFly 3 application or starting fresh with the latest and greatest PatternFly design system, this document is intended to help provide designers with information regarding resources and support available.
+Whether you are converting from a previous PatternFly application or starting fresh with the latest and greatest PatternFly design system, this page is intended to help provide designers with information regarding resources and support available.
 
-## Getting the design kit
+## Designing with PatternFly
+PatternFly offers a variety of resources to help you make the most of the design system, including design guidelines for style, content, and component usage, and behavior. You can use these assets in combination with the design kit to create consistent and usable product experiences.
 
-The PatternFly design kit was originally developed for [Sketch users](#sketch-users), but is now available for use through [any design tool](#using-other-design-tools).
+### Design kit
+The design kit is a symbol library implemented in Sketch to help designers like you contribute to PatternFly.
 
-### Sketch users
+[**What's in the design kit?**](/v4/get-started/design#whats-in-the-design-kit) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
 
-#### Before you begin
-The design kit is a collection of Sketch assets that make it easy for designers to create hi-fi mockups that accurately represent PatternFly components and layouts. It works with Sketch libraries. To use it, you need to install [Sketch version 68](https://www.sketch.com/) or above. 
 
-#### Download the font
-1. PatternFly currently uses Red Hat Text and Red Hat Display. To download these fonts, go to https://github.com/RedHatOfficial/RedHatFont and download the whole repository. Then select all files in the "TTF" folder and double-click. This will install them onto your computer locally.
 
-#### Download the template file
-2. Access the [PatternFly template file](https://www.sketch.com/s/729c2eee-e8b6-4fcd-8a79-f6faa8c30f89) on Sketch cloud.
-3. Click **Download Document** to open the file. Then, click File → Save as template.
+### Style guidelines
+What typography should you be using? What about colors? Spacing? Check out styles to learn everything you need to know about our foundational design elements.
+
+[**View styles**](/guidelines/colors) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
 
-#### Add the symbol library
-4. Access the [PatternFly design library](https://www.sketch.com/s/2cf1063b-5283-4e0b-b8a6-cbb1ac07e29e) on Sketch cloud.
-5. Click **Add library to Sketch** on the bottom right to add the library. You'll be prompted with a modal. Click **Add library to Sketch** again. Sketch will show you the library in your preferences window.
+### Usage and behavior guidelines
```

Comment:
I feel like this paragraph should change. It's left over from when we kept design guidelines in a separate location on the site. Maybe try?

Should you use a wizard or a form to solve this problem? How do you write a good button label? Which navigation makes the most sense? Get answers by clicking the Design guidelines tab associated with our component pages.

Reply:
@mcarrano Made those updates!